src/app/migration/src/src/app/modules/core/services/user/user.service.ts

```diff
@@ -43,12 +44,28 @@ export class UserService {
    * reference of lerner service.
    */
   public learner: LearnerService;
+  /**
+ * Reference of appId
+ */
+  public _appId: string;
+  /**
+   * Reference of channel
+   */
+  public channel: string;
+  /**
+   * Reference of dims
+   */
+  public dims: Array<string> = [];
```

Comment:
dims not assigned value in user service but referred in many components

Reply:
done